CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - Request recipes can now be organized into folders ([#60](https://github.com/LucasPickering/slumber/issues/60))
   - See [the docs](https://slumber.lucaspickering.me/book/api/request_collection/request_recipe.html#folder-fields) for usage examples
 - Add `slumber show config` and `slumber show collection` subcommands
+- Support booleans and numbers for query values ([#141](https://github.com/LucasPickering/slumber/issues/141))
```

Comment:
This should be under the Top `Unreleased` section, you have it under 0.18.0. Maybe you added it before the 0.18 release and it stayed here during a rebase? Feel free to add a `Added` section to `Unreleased` for this.